daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -64,6 +64,17 @@ object Speedy {
     def popEnv(count: Int): Unit =
       env.subList(env.size - count, env.size).clear
 
+    def stackTrace(): ArrayList[Location] = {
```

Comment:
Afaict, the only thing you are doing with this is to call `asScala` on it so I would suggest to either work with whatever the Scala type is called directly or remove the conversion.

Reply:
@remyhaemmerle-da Can you please advise the correct Scala type to use here.

Reply:
`ImmArray` or `ImmArraySeq` from the `//daml-lf/data` library; the latter if you want Scala collections operations directly, former otherwise.

Reply:
I went for `ImmArray`. I don't know if that's the right choice.